engine-tests/src/main/java/org/terasology/TerasologyTestingEnvironment.java

```diff
@@ -134,7 +134,6 @@ public static void tearDown() throws Exception {
         env.close();
     }
 
-
```

Comment:
add this line back in so you don't make extra changes

Reply:
:roll_eyes: if you insist...